pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceBaseArguments.java

```diff
@@ -49,6 +49,31 @@ public abstract class PerformanceBaseArguments extends CmdBase{
             descriptionKey = "tlsTrustCertsFilePath")
     public String tlsTrustCertsFilePath = "";
 
+    @Option(names = {"--use-keystore-tls" }, description = "Use KeyStore TLS", descriptionKey = "useKeyStoreTls")
+    public Boolean useKeyStoreTls = null;
```

Comment:
I have updated it based on your suggestion. Using primitive types was also my intuition, but all other booleans in this class are using a boolean object, so I thought it would be good for consistency. Do you think, the other booleans for tlsAllowInsecureConnection and tlsHostnameVerificationEnable should also be changed to primitive type to be able to get rid of the null checks?

Reply:
We just need to focus on this value. Otherwise, it may change a lot of code.